packages/@aws-cdk/aws-cloudfront/lib/origin_access_identity.ts

```diff
@@ -0,0 +1,110 @@
+import iam = require('@aws-cdk/aws-iam');
+import cdk = require('@aws-cdk/core');
+import { CfnCloudFrontOriginAccessIdentity } from './cloudfront.generated';
+
+export interface CloudFrontOriginAccessIdentityProps {
```

Comment:
Should be called OriginAccessIdentity (no CloudFront prefix). I know this module has a lot of these, but those are non-conventional and will be changed. Let's build this one per our guidelines.